competitions/utils.py

```diff
@@ -1,12 +1,32 @@
-from typing import Optional
+from typing import List, Optional
 
 import constants
-from competitions.data import Competition, CompetitionId
+from competitions.data import Competition, CompetitionId, ModelConstraints
 
 
-def get_competition(id: CompetitionId) -> Optional[Competition]:
-    """Returns the competition with the given id, or None if it does not exist."""
-    for x in constants.COMPETITION_SCHEDULE:
-        if x.id == id:
-            return x
+def get_model_constraints(id: CompetitionId) -> Optional[ModelConstraints]:
+    """Returns the model constraints for the given id, or None if it does not exist."""
+    if id in constants.MODEL_CONSTRAINTS_BY_COMPETITION_ID:
```

Comment:
No change needed, just FYI. This is equivalent to constants. MODEL_CONSTRAINTS_BY_COMPETITION_ID.get(id, None)

Reply:
Good call, made the change.